lib/fs.js

```diff
@@ -520,7 +520,6 @@ function tryStatSync(fd, isUserFd) {
   } finally {
```

Comment:
I guess it was written like this for performance. If not, isn't cleaner to use `catch` and get rid of the threw variable?